grass/process.c

```diff
@@ -28,6 +28,9 @@ int proc_alloc() {
         if (proc_set[i].status == PROC_UNUSED) {
             proc_set[i].pid = ++proc_nprocs;
             proc_set[i].status = PROC_LOADING;
+            
+            memset(&proc_set[i].sc, 0, sizeof(struct syscall));
+            proc_set[i].sc.type = SYS_UNUSED;
```

Comment:
I feel like this line is not necessary since the `memset` above should have already set `sc.type` to 0, i.e., `SYS_UNUSED`.

Reply:
Removed both, as `proc_try_syscall` is only run once a process makes a system call and moves through `excp_entry()`, which will overwrite the PCB syscall with the user space syscall. Also removed `proc->sc.type = SYS_UNUSED` at the end of `proc_try_syscall()` for a similar reason.

Let me know whether there may be something that was not considered.